VotingApplication/VotingApplication.Web/Scripts/Controllers/ResultsPageController.js

```diff
@@ -53,9 +60,6 @@
             if (dataUnchanged)
                 return;
 
-            // Hack to fix insight's lack of data reloading
```

Comment:
We can get rid of all the dataUnchanged logic now and just rely on the backend sending 304.
This should also mean that barCount = data.length